src/test/java/com/automationexercise/tests/Tests/TestCasesPageTest.java

```diff
@@ -1,37 +1,38 @@
 package com.automationexercise.tests.Tests;
 
 import com.automationexercise.DriverManager.BrowserFactory;
-import com.automationexercise.DriverManager.DriverManager;
 import com.automationexercise.Pages.HomePage;
 import com.automationexercise.Pages.TestCasesPage;
-import com.automationexercise.Utilities.Utility;
 import org.openqa.selenium.WebDriver;
 import org.testng.Assert;
 import org.testng.annotations.*;
+import org.testng.asserts.SoftAssert;
 
 import static com.automationexercise.DriverManager.DriverFactory.createInstance;
 import static com.automationexercise.DriverManager.DriverManager.*;
 import static com.automationexercise.Utilities.Utility.openWebsite;
 
 public class TestCasesPageTest  {
-    WebDriver driver = BrowserFactory.CHROME.createDriver();
     @BeforeMethod(alwaysRun = true)
     public void setUp() {
-        /*createInstance("chrome");
-        openWebsite("https://automationexercise.com/");*/
-        driver.get("https://automationexercise.com/");
+        createInstance("chrome");
```

Comment:
Read the browser field from **environment.properties** instead of using it hard coded.